app/src/main/res/layout/activity_chat.xml

```diff
@@ -154,7 +154,6 @@
             app:outcomingTextLinkColor="@color/high_emphasis_text"
             app:outcomingTextSize="@dimen/chat_text_size"
             app:outcomingTimeTextSize="12sp"
-            app:textAutoLink="all"
```

Comment:
removing 
`app:textAutoLink="all"`
works and may be mandatory to fix the bug.

However this will change the behavior of all ChatMessage types and automatic link detection won't work as described in
https://github.com/nextcloud/talk-android/pull/3481#issuecomment-2099087795